### DISCUSSION TITLE
If i press F5 the game switch between Fullscreen and Windowmode.

### DISCUSSION THREAD

**Discussion by V4ldaran** [Lailloken/Exile-UI#424] [General]
How can i disable that?

**Comment by Lailloken** [comment]
You're only the second person to report this issue. I don't know why this happens when you press F5 (this key doesn't do anything in my tool), but switching between window-mode is a system-wide Windows feature activated by pressing ALT+ENTER.

Since the F5 hideout macro sends ENTER at the end, the ALT state on your keyboard may be stuck, or something else is causing an ALT key-press. What happens if you tap ALT multiple times (just to clear the key-state) and then press F5?

**Reply by V4ldaran** [reply]
Unfortunately pressing the ALT Key multiple times didnt fixed it. But its definitely the Hideout Macro because its only happens if i have awakened poe trade open at the same time.

**Reply by Lailloken** [reply]
As a workaround, you can disable it in Awakened and instead create a macro in my tool:
- open the `modules` folder inside your Lailloken UI folder

- make a copy of the `hotkeys.ahk` file

- rename the copy to `hotkeys custom.ahk`

- right-click the new file, click edit

- delete everything inside, then paste this:

```
#IfWinActive ahk_group poe_window

F5::
Clipboard := "/hideout"
ClipWait, 0.1
SendInput, {Enter}^{v}{Enter}
KeyWait, F5
Return

#If
```

- save the file, and restart the tool

**Reply by V4ldaran** [reply]
Oh thanks, thats great!

**Comment by V4ldaran** [comment]
Okay its not just the HO Macro, if im using MercuryTrade together with this Tool its also does weird things if i click on the invite button and so on.

**Reply by Lailloken** [reply]
Is it the ALT+ENTER thing again?
Which keyboard layout/language are you using?

EDIT: Also, which AutoHotkey version are you using? You can check by right-clicking the L-icon in the system tray and clicking "help".

**Reply by V4ldaran** [reply]
Im using the German Layout. Version is: 1.1.37.02

**Reply by Lailloken** [reply]
Can you test if something changes if you load my tool first/last?
Also, does it happen immediately after launching my tool, or does it happen after having it run for a while?

**Reply by V4ldaran** [reply]
I just tried it bothways and it makes no difference, and it happens immediately.

**Reply by Lailloken** [reply]
I tried to research this issue today but I am none the wiser. Apparently, this issue has existed for a long time, is inconsistent, and happens due to incompatibility between multiple applications that manage/create hotkeys. It would be helpful if I could recreate it on my end.

I'll try again tomorrow, but it doesn't look promising right now.

**Reply by V4ldaran** [reply]
I used the same programs last league without problems though, maybe i should try an older version of your tool?

**Reply by Lailloken** [reply]
You could try that, yes. Make a backup of your script folder, and then you can use the built-in updater to go back one version at a time.

**Reply by V4ldaran** [reply]
Okay i just tried it, unfortunately it didnt helped.
